src/hw1/bash_commands.py

```diff
@@ -0,0 +1,35 @@
+def head(file, strings=10):
+    for i in range(strings):
+        print(file.readline(), end="")
+
+
+def tail(file, strings=10):
+    lines = [line for line in file]
+    lines.reverse()
+    for i in range(strings):
+        print(lines[i], end="")
+
+
+def wc(file):
+    wordCount = 0
+    symbolCount = 0
+    lines = [line for line in file]
+    lineCount = len(lines)
+    for line in lines:
+        words = line.split(" ")
+        if "\n" in words:
+            words.remove("\n")
+        wordCount += len(words)
+        symbolCount += len(line) - 1
+    print(str(lineCount) + " " + str(wordCount) + " " + str(symbolCount))
```

Comment:
Such writing is very difficult to understand. The cool way is to write `print(f"{lineCount} {wordCount} {symbolCount}")`